wxtbx/metallicbutton.py

```diff
@@ -78,11 +81,15 @@ def __init__ (self,
     # XXX this crashes on wxOSX_Cocoa!
     if (not 'wxOSX-cocoa' in wx.PlatformInfo) :
       self._label2_font.SetStyle(wx.FONTSTYLE_ITALIC)
+      # with wx4c.set_font_style(wx.FONTSTYLE_ITALIC) as fs:
+      #   self._label2_font.SetStyle(fs)
     font_size = label_size
     self._label_font = self.GetFont()
     self._label_font.SetPointSize(label_size)
     if style & MB_STYLE_BOLD_LABEL :
-      self._label_font.SetWeight(wx.FONTWEIGHT_BOLD)
+      self._label2_font.SetWeight(wx.FONTWEIGHT_BOLD)
```

Comment:
Should this be `self._label_font` instead of `self._label2_font`? On the Phenix GUI, this causes the description text in the application list to be set to bold (e.g. for phenix.refine, the "Automated refinement using diffraction data..." text is set to bold). It also prevents the button text from being bold (e.g. the "Open in Coot" button text is not bold). Other than that, things look fine. I'll double-check with Nick before merging.